test/Feature/WaveOps/WaveActiveCountBits.test

```diff
@@ -6,16 +6,12 @@ RWStructuredBuffer<uint> Out : register(u1);
 void main(uint3 threadID : SV_DispatchThreadID) {
   bool B1 = false;
 
-  switch (value[threadID.x]) {
-    case 0: // threads 0 and 1; result is number of active lanes (2)
-      Out[threadID.x + 4] = WaveActiveCountBits(true); // threads 0 and 1
-    case 2:
-      B1 = true; // set b1 to true for thread 3
-      break;
-    default:
-      Out[threadID.x + 4] = WaveActiveCountBits(false); // thread 2; expect 0
-      break;
-  }
+  if (value[threadID.x] == 0)
+    Out[threadID.x + 4] = WaveActiveCountBits(true); // threads 0 and 1
+  if (value[threadID.x] == 0 || value[threadID.x] == 1)
+    B1 = true; // set b1 to true for thread 3
+  else
+    Out[threadID.x + 4] = WaveActiveCountBits(false); // thread 2; expect 0
```

Comment:
This writes output for thread 3, since it's the only thread not going through the main `if` branch.

Incidentally, the expected outputs are also truncated, excluding this output, while it includes an expected `0` output for thread 2, which was never written from the shader (which passes due to zero-initialization of output buffer).